src/lib/components/ui/RecentCategoryArticles.svelte

```
@@ -1,137 +1,151 @@
 
-    const categoryOrder = [
-        'Layer 1',
-        'Layer 2',
-        'Interoperability',
-        'DeFi',
-        'Privacy',
-        'DePIN',
-        'Infrastructure'
-    ];
+	const {
+		articlesPerCategory = 1,
```

ESLint

[error] Identifier 'articlesPerCategory' has already been declared(js_parse_error)

(svelte/valid-compile)